### PR TITLE
Refactor labels + stuff

### PR DESCRIPTION
**Pull request by Eirika**

### CODE REVIEW

File: player/views/player/models.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+models.py
```
Comment (Amoki) [inline player/views/player/models.py]: ??

File: music/helpers/youtube.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ def get_info(ids):
         detailedVideo = {
             'id': detail["id"],
             'name': detail["snippet"]["title"],
+            'channel_name': detail["snippet"]["channelTitle"],
```
Comment (Amoki) [inline music/helpers/youtube.py]: i love it :D

File: music/models.py [hunk 3]
```diff
@@ -36,6 +36,7 @@ def __unicode__(self):
 class TemporaryMusic(models.Model):
     music_id = models.CharField(max_length=16)
     name = models.CharField(max_length=255)
+    channel_name = models.CharField(max_length=255, null=True)
```
Comment (Amoki) [inline music/models.py]: remove null=True and set '' as value in migration

Comment (Eirika) [inline music/models.py]: Set '' as a default value ? field=models.CharField(max_length=255, default='')  ?

Comment (Amoki) [inline music/models.py]: No,field=models.CharField(max_length=255) and in the migration model, you can force a default value for old entries that were in database before.
manage.py makemigration asks you this if you didn't set null=True

**Pull request merged**